apps/docs/content/guides/self-hosting/docker.mdx

```diff
@@ -159,6 +159,13 @@ You'll want to update the Studio(Dashboard) frequently to get the latest feature
 
 While we provided you with some example secrets for getting started, you should NEVER deploy your Supabase setup using the defaults we have provided. Follow all of the steps in this section to ensure you have a secure setup, and then [restart all services](#restarting-all-services) to pick up the changes.
 
+<Admonition type="danger">
+
+Your instance is running without any type of encryption.
+This guide does not walk you through how to secure your instance using SSL (`https://...`). Before exposing your instance on the Internet, make sure all traffic is secure. This is outside of the scope for this guide for the time being.
+</Admonition>
+
+
```

Comment:
this would fit better under the existing warning about securing services: https://github.com/supabase/supabase/pull/34216/files#diff-3552d44e92efffffff97b64076ea318ab06f07980d335d620f2bfe5447167135R90-R94

```
Your app is now running with default credentials and http endpoints do not have TLS configured.
```

We are discussing adding TLS configuration guidance to the `Securing your services` section.

Reply:
I don't quite follow - this warning *is* under the existing warning about securing services, isn't it?

Reply:
Rewording, "under" here refers to being within the same warning block. 

The warning is about the risk of deploying directly with defaults and directs you to a section about securing your setup. Thus it makes sense adding additional security+defaults language in the same location.

Reply:
The current warning block is under the heading **Installing and running Supabase**.
I don't think the warning I've proposed to add belongs under that heading, but rather under the heading **Securing your services**.

It's misleading to have a complete section called **Securing your services** and bluntly talk about changing the default credentials, while leaving it to the user to communicate the very same keys in plain text... Why even go trough the trouble of changing the defaults if you're gonna show them to world anyway.

I still believe my edit is more appropriately placed, including the repetition.